src/hooks/useEmptyViewHeaderHeight/index.ios.ts

```diff
@@ -0,0 +1,10 @@
+import useSafeAreaInsets from '@hooks/useSafeAreaInsets';
+import {BUTTON_HEIGHT, BUTTON_MARGIN, HEADER_HEIGHT} from './CONST';
+
+function useEmptyViewHeaderHeight(isSmallScreenWidth: boolean): number {
+    const safeAreaInsets = useSafeAreaInsets();
+
+    return isSmallScreenWidth ? HEADER_HEIGHT + BUTTON_HEIGHT + BUTTON_MARGIN + safeAreaInsets.top : HEADER_HEIGHT;
```

Comment:
This file is specific to iOS, so I believe isSmallScreenWidth will always be true

Reply:
not for iPads I guess